source/D2Game/include/GAME/Game.h

```diff
@@ -60,20 +62,20 @@ static const HGAMEDATA D2GameReservedSlotHandle = GetGameHandleFromHashValue(D2G
 
 
 using FnCloseGame = void(__fastcall*)(WORD nGameId, uint32_t nProductCode, uint32_t nSpawnedPlayers, int32_t nFrame);
-using FnLeaveGame = void(__fastcall*)(D2ClientInfoStrc** ppClientInfo, WORD nGameId, int32_t nClassId, int32_t nLevel, uint32_t nExperience, int32_t a6, uint32_t nFlags, const char* szCharName, const char* a9, int32_t bUnlockChar, int32_t nZero11, int32_t nZero12, const char* szAccountName, int32_t a14, FILETIME* pSaveCreationTimestamp); //TODO: Unknown args
+using FnLeaveGame = void(__fastcall*)(D2ClientInfoStrc** ppClientInfo, WORD nGameId, int32_t nClassId, int32_t nLevel, uint32_t nExperienceLow, int32_t nExperienceHigh, uint32_t nFlags, const char* szCharName, D2CharacterPreviewInfoStrc* pPreviewInfoStrc, int32_t bUnlockChar, int32_t nZero11, int32_t nZero12, const char* szAccountName, int32_t nCharSaveTransactionToken, FILETIME* pSaveCreationTimestamp);
```

Comment:
(on the phone at the moment so limited ability to paste links)

There are two places where experience spans two variables, in the other place high is explicitly set to 0, suggesting it goes over two variables, rather than always a uint32 split over two variables.

Reply:
> in the other place high is explicitly set to 0
This is just a 32bit value being widened to 64bit as per the MSVC x86 ABI: https://godbolt.org/z/e356aqrhq